miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniAppSdkConfig.kt

```diff
@@ -13,11 +13,10 @@ data class MiniAppSdkConfig(
     val baseUrl: String,
     val rasAppId: String,
     val subscriptionKey: String,
-    val hostAppVersionId: String,
```

Comment:
So, we can remove ` * @property hostAppVersionId Version of the host app, used to determine feature compatibility for Mini App.` comment in this file?